README.md

```diff
@@ -24,22 +27,26 @@ This tap:
 ## Streams
 
 **[export](https://developer.mixpanel.com/docs/exporting-raw-data#section-export-api-reference)**
-- Endpoint: https://data.mixpanel.com/api/2.0/export
+- Standard Server endpoint: https://data.mixpanel.com/api/2.0/export
+- EU Residency Server endpoint: https://data-eu.mixpanel.com/api/2.0/export
 - Primary key fields: `event`, `time`, `distinct_id`
 - Replication strategy: INCREMENTAL (query filtered)
   - Bookmark: `time`
   - Bookmark query field: `from_date`, `to_date`
 - Transformations: De-nest `properties` to root-level, re-name properties with leading `$...` to `mp_reserved_...`, convert datetimes from project timezone to UTC.
 
 **[engage](https://developer.mixpanel.com/docs/data-export-api#section-engage)**
-- Endpoint: https://mixpanel.com/api/2.0/engage
+  - Standard Server endpoint: https://mixpanel.com/api/2.0/engage
+  - EU Residency Server endpoint: https://mixpanel.com/api/2.0/engage
```

Comment:
EU endpoint will be: https://eu.mixpanel.com/api/2.0/engage

Reply:
Updated EU endpoint.